Samples/kmp-app/androidApp/build.gradle.kts

```diff
@@ -0,0 +1,27 @@
+plugins {
+    id("com.android.application")
+    kotlin("android")
+}
+
+android {
+    compileSdk = 32
+    defaultConfig {
+        applicationId = "sample.kpm_app.android"
+        minSdk = 30
+        targetSdk = 32
+        versionCode = 1
+        versionName = "1.0"
+    }
+    buildTypes {
+        getByName("release") {
+            isMinifyEnabled = false
```

Comment:
Any reason why is it `isMinifyEnabled`?

Reply:
I'm not sure, this was one of the things that was also included in the newest branch of the experimental project, should I remove it?

Reply:
It's fine, but later we've to enable it and test it out, this is the boolean that enables minification of sourcecode, etc.